xtask/src/internal.rs

```diff
@@ -178,8 +178,6 @@ pub struct BuildClient {
 pub enum ServerVariant {
```

Comment:
Are we expecting any other server variants in the future?

(Something something baremetal?)

Reply:
Yeah I'm leaving it in for now, so that we can e.g. use it to select between multiple trusted runtimes, but it may well be that we won't actually need it, in which case I'll delete it later on.